ambient.d.ts

```diff
@@ -24,11 +24,204 @@
+      THIRDPARTY_TRANSACTIONS_AUTHORIZATIONS_POST: FspEndpointTypesEnum.THIRDPARTY_TRANSACTIONS_AUTHORIZATIONS_POST;
+      THIRDPARTY_TRANSACTIONS_AUTHORIZATIONS_PUT: FspEndpointTypesEnum.THIRDPARTY_TRANSACTIONS_AUTHORIZATIONS_PUT;
+      THIRDPARTY_TRANSACTIONS_AUTHORIZATIONS_ERROR: FspEndpointTypesEnum.THIRDPARTY_TRANSACTIONS_AUTHORIZATIONS_ERROR;
+      THIRDPARTY_CALLBACK_URL_TRX_REQ_POST: FspEndpointTypesEnum.THIRDPARTY_CALLBACK_URL_TRX_REQ_POST;
+    };
+    FspEndpointTemplates: {
+      TRANSACTION_REQUEST_POST: string;
+      TRANSACTION_REQUEST_PUT: string;
+      TRANSACTION_REQUEST_GET: string;
+      TRANSACTION_REQUEST_PUT_ERROR: string;
+      PARTICIPANT_ENDPOINTS_GET: string;
+      PARTICIPANTS_GET: string;
+      PARTIES_GET: string;
+      PARTIES_PUT_ERROR: string;
+      PARTIES_SUB_ID_PUT_ERROR: string;
+      ORACLE_PARTICIPANTS_TYPE_ID: string;
+      ORACLE_PARTICIPANTS_TYPE_ID_CURRENCY: string;
+      ORACLE_PARTICIPANTS_TYPE_ID_SUB_ID: string;
+      ORACLE_PARTICIPANTS_TYPE_ID_CURRENCY_SUB_ID: string;
+      ORACLE_PARTICIPANTS_BATCH: string;
+      TRANSFERS_POST: string;
+      TRANSFERS_PUT: string;
+      TRANSFERS_PUT_ERROR: string;
+      BULK_TRANSFERS_POST: string;
+      BULK_TRANSFERS_PUT: string;
+      BULK_TRANSFERS_PUT_ERROR: string;
+      THIRDPARTY_TRANSACTION_REQUEST_PUT_ERROR: string;
+      THIRDPARTY_TRANSACTION_REQUEST_POST: string;
+    };
+  }
+  interface Enum {
+    Http: HttpEnum;
+    EndPoints: EndPointsEnum;
+    Events: {
+      Event: {
+        Action: {
+          POST: string;
+        };
+        Type: {
+          TRANSACTION_REQUEST: string;
+        };
+      };
+    };
+  }
+  class Endpoints {
+    fetchEndpoints(fspId: string): Promise<any>
+    getEndpoint(switchUrl: string, fsp: string, endpointType: FspEndpointTypesEnum, options?: any): Promise<string>
+    initializeCache(policyOptions: object): Promise<boolean>
+  }
+
+  class Request {
+    sendRequest(url: string, headers: any, source: string, destination: string, method?: string, payload?: any, responseType?: string, span?: any, jwsSigner?: any): Promise<any>
+  }
+
+  interface Util {
+    Endpoints: Endpoints;
+    Request: Request;
+    Hapi: any;
+    OpenapiBackend: any;
+  }
+
+  const Enum: Enum
+  const Util: Util
+  const HealthCheck: any
+}
+
+declare module '@mojaloop/central-services-error-handling' {
+  class FSPIOPError {
+    toApiErrorObject(includeCauseExtension?: boolean, truncateExtensions?: boolean): any
+    apiErrorCode: {
+      code: number;
+      message: string;
+    }
+
+    message: string
+  }
+  const Factory: {
+    FSPIOPError: FSPIOPError;
+  }
+  const Enums: {
+    FSPIOPErrorCodes: {
+      DESTINATION_FSP_ERROR: any;
+      DESTINATION_COMMUNICATION_ERROR: any;
+    };
+  }
   export function validateRoutes(options?: object): object;
+  export function ReformatFSPIOPError(error: any, apiErrorCode?: any, replyTo?: any, extensions?: any): any
```

Comment:
you need return type for this function because you are using it `src/domain/thirdpartyRequests/transactions.ts`